wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -498,6 +500,16 @@ public static class Config extends PluginConfig {
     @Macro
     private String precondition;
 
+    @Name(NAME_PRECONDITION_SQL)
+    @Description("Precondition expression specifying filtering before applying directives (true to filter)")
+    @Macro
+    private String preconditionSql;
+
+    @Name(NAME_PRECONDITION_TOGGLE)
+    @Description("Precondition toggle to toggle between JEXL and SQL")
+    @Macro
+    private String preconditionToggle;
```

Comment:
This field should be called something like "Precondition Format" or "Precondition Language".

Reply:
Can you rename the field to `preconditionFormat`? I don't think we use the word toggle anywhere else, and this is effectively a selector.